synapse/lib/storm.py

```diff
@@ -924,18 +920,6 @@ def _stormOperFilt(self, query, oper):
 
         [query.add(t) for t in query.take() if cmpr(t)]
 
-    def _stormOperOr(self, query, oper):
```

Comment:
Neither the `or` or `and` operator were hooked up to anything.

Reply:
Yep, good call.  Vestigial code from the old ( non-combinable ) way logic operators worked.